plugin/src/main/groovy/com/novoda/gradle/command/AndroidCommandPluginExtension.groovy

```diff
@@ -83,6 +86,14 @@ public class AndroidCommandPluginExtension {
         monkey
     }
 
+    void script(Action<NamedDomainObjectContainer<InputSpec>> action) {
```

Comment:
Oh, and about adding the container you don't need to do anything special, just make the `scripts` field public in the extension (in BPP we want to add it to the `project` we have no extension there)